lib/repl.js

```diff
@@ -545,6 +545,12 @@ REPLServer.prototype.createContext = function() {
     });
   });
 
+  // Refer: https://github.com/nodejs/node/pull/3729#issuecomment-155460861
+  // REPL stores the result of the last evaluated expression in `context`'s `_`.
```

Comment:
Can you remove the markdown formatting, as it doesn't do anything. Also, the uses of "REPL" should be "the REPL"